CollAction/Controllers/ProjectsController.cs

```diff
@@ -373,79 +373,6 @@ public async Task<IActionResult> SendProjectEmailPerform([Bind("ProjectId", "Sub
             return RedirectToAction(nameof(ManageController.Index), "Manage");
         }
 
-        [HttpGet]
-        public async Task<IActionResult> ChangeSubscriptionFromToken(ChangeSubscriptionFromTokenViewModel unsubscribeViewmodel)
```

Comment:
Some of these are needed to unsubscribe from email, careful... All the "FromToken" variants are used to have users unsubscribe from e-mail.

Reply:
Good catch! I searched the front-end for where this was used; didn't find the link in the e-mail text. Restored the FromToken files, and removed only the FromAccount methods.